test/jdk/jdk/security/jarsigner/Spec.java

```diff
@@ -125,7 +125,9 @@ public static void main(String[] args) throws Exception {
         iae(()->b1.setProperty("internalsf", "Hello"));
         npe(()->b1.setProperty("sectionsonly", null));
         iae(()->b1.setProperty("sectionsonly", "OK"));
-        npe(()->b1.setProperty("altsigner", null));
```

Comment:
Is 'altsigner' support removed? But I saw it being used in later part of this test. The javadoc for JarSigner.Builder only lists a subset of above properties. Are those not in javadoc internal and can be removed any time, just curious?

Reply:
It's deprecated. I remember we are already thinking about deprecating the altSign mechanism when creating the JarSigner API and therefore we didn't add them to the spec from the beginning.

Bug id added.